### ISSUE TITLE
slimefun.cargo.bypass is not checked

### ISSUE THREAD

**Issue by runelynx** [TheBusyBiscuit/Slimefun4#79]
## Description
	slimefun.cargo.bypass  is not checked.  Its supposed to allow players who do not own cargo nodes to edit them anyway.  But players without that permission can do it still! Note that the players also do not have slimefun.inventory.bypass. 

## Server Log
There are no log entries related to this. 

## Environment
1.11.2 Minecraft and the latest CoreLib and Slimefun currently available.

**Comment by TheBusyBiscuit** [comment]
Lack of Information

**Comment by runelynx** [comment]
We walled off the area and abandoned the idea of showcasing the functionality of cargo nodes.

For your info... With the info I gave above, and in all of 10 seconds, I was able to find this in your code. Basically you override the permission if the user has access to the end chest...   i.e. you don't allow for the scenario that someone can access the chest but not the cargo nodes. Your lack of interest in helping is a turn off. 👎 

All you had to say was the permission is not meant to lock cargo nodes... the protections on the chest attached to the node is supposed to do that. 

src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/CargoOutputNode.java 
Line 112
CSCoreLib.getLib().getProtectionManager().canAccessChest(p.getUniqueId(), b) || p.hasPermission("slimefun.cargo.bypass");
